tests/plantcv/transform/test_auto_correct_color.py

```diff
@@ -10,3 +11,11 @@ def test_auto_correct_color(transform_test_data):
     rgb_img = cv2.imread(transform_test_data.colorcard_img)
     corrected_img = auto_correct_color(rgb_img=rgb_img)
     assert np.shape(corrected_img) == np.shape(rgb_img) and np.sum(corrected_img) != np.sum(rgb_img)
+
+def test_auto_correct_color_subset(transform_test_data):
+    """Test for PlantCV."""
+    # Load rgb image
+    rgb_img = cv2.imread(transform_test_data.colorcard_img)
+    roi = rectangle(rgb_img, x = 750, y = 250, h = 1000, w = 750)
```

Comment:
This is nit-picky, and not consistently upheld across plantcv CI testing, but best practice would be to use only one plantcv function/algorithm per test. It takes additional lines of code but we can manually define the ROI, as is done in /plantcv/tests/plantcv/roi/test_quick_filter.py .